parser.y

```diff
@@ -1961,6 +1995,15 @@ ConstraintElem:
 			Refer:		$8.(*ast.ReferenceDef),
 		}
 	}
+|	"CHECK" '(' Expression ')' EnforcedOrNotOpt
+	{
+		$$ = &ast.Constraint{
+			Tp:		ast.ConstraintCheck,
+			Expr:		$3.(ast.ExprNode),
+                }
```

Comment:
Reindent this line using tabs.

Also, the `ENFORCED`/`NOT ENFORCED` should be included (the default is `ENFORCED`) in the Restore() text. On MySQL it is restored as `/*!80016 NOT ENFORCED */`.

Reply:
done